libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -784,10 +784,32 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "media_video_cant_optimize";
             case EDITOR_PUBLISHED_POST:
                 return "editor_post_published";
+            case EDITOR_POST_PUBLISH_TAPPED:
+                return "editor_post_publish_tapped";
+            case EDITOR_POST_SCHEDULED:
+                return "editor_post_scheduled";
+            case EDITOR_POST_VISIBILITY_CHANGED:
+                return "editor_post_visibility_changed";
+            case EDITOR_POST_TAGS_ADDED:
+                return "editor_post_tags_added";
+            case EDITOR_POST_PUBLISH_NOW_TAPPED:
+                return "editor_post_publish_now_tapped";
+            case EDITOR_POST_PASSWORD_CHANGED:
+                return "editor_post_password_changed";
             case EDITOR_UPDATED_POST:
                 return "editor_post_updated";
             case EDITOR_SCHEDULED_POST:
```

Comment:
I realized that this event exists after I added the `EDITOR_POST_SCHEDULED` above. The difference is that `EDITOR_SCHEDULED_POST` is being tracked in the `UploadService` when an upload actually takes place. It seems like a duplicate so I want to hear some thoughts about it. Thanks.

Reply:
On iOS, I am seeing a class that contains the recent events that were added https://github.com/wordpress-mobile/WordPress-iOS/blob/c9f0dbaca94a117ed003e07216ca9510705aa069/WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

I am not seeing any other post published events. 

`EDITOR_POST_SCHEDULE_CHANGED` 
I was thinking of using this name as it implies the post's schedule was changed and the existing `EDITOR_POST_SCHEDULED` is like the terminal operation; i.e the post is actually scheduled when the scheduling action takes place. Let me know your thoughts.

Reply:
Hi @ashiagr I discussed this with the rest of the team and we agreed on changing this. Resolved here 86c98c0